lint/util.py

```diff
@@ -336,7 +336,7 @@ def tmpfile(cmd, code, filename, suffix='', output_stream=STREAM_STDOUT, env=Non
     if suffix:
         filename = os.path.splitext(filename)[0] + suffix
 
-    file = tempfile.NamedTemporaryFile(delete=False)
+    file = tempfile.NamedTemporaryFile(delete=False, suffix=suffix)
```

Comment:
Bc this is a regression from https://github.com/SublimeLinter/SublimeLinter/commit/a3db729367c0ceb423a3f617ab3fa8ab4b739409

(we just forgot about the suffix completely) This line alone should restore the old SublimeLinter behavior. So if you do just that and it works again. I could merge immediately bc I don't have to think through this whole thing. 

I don't remember we touched `get_tempfile_suffix` recently (git blame?). So I don't want to start a change here for what appears to be hotfixable. WDYT?

Of course the initial lines about `filename`in that function are no-ops. `filename` is not used anymore. Say, we had a bad day here.

Reply:
Damn, who tests these things. And why does git `blame` me for these lines? 😒

The hotfix (and removal of those lines about "filename") seems like the way to go to me. Can someone confirm the fix for the php thing?